docs/rst/notes/humble/previous_versions/v2.0.4.rst

```diff
@@ -0,0 +1,55 @@
+Humble Hierro (v2.0.4)
+----------------------
```

Comment:
This title requires a lower title level. I think `^^^^^^^^` is the one required. Right now, the index is disordered.